setup.py

```diff
@@ -3,8 +3,15 @@
 from _about import __version__
 import setuptools
 
-with open("README.md", "r",encoding="utf8") as fh:
-    long_description = fh.read()
+vers = sys.version_info.major
+if vers == 2:
+    with open("README.md", "r") as fh:
+        long_description = fh.read()
+else:
+    with open("README.md", "r",encoding="utf8") as fh:
+        long_description = fh.read()
+
+requires = ">=2, <3" if vers == 2 else ">=3"
```

Comment:
I am not sure I understand the point of this.

Reply:
It's related to the line `python_requires='>=3',` inside the setuptools.setup() function. When that line is present, the build will fail on the "Install package" step, with the error `ERROR: Package 'pycgm' requires a different Python: 2.7.18 not in '>=3'`. This is because the package is being told that it requires a python version of at least 3.0, when it's running on 2.7. My change avoids this error (and doesn't appear to cause any issues because the rest of the workflow passes), but if there's a better way to handle the error then we can do so instead.

Reply:
what is wrong with `python_requires='2.7, >=3'` ?

Reply:
Commas in python_requires work as a logical and, so `>=2.7, >=3.6` would still be an issue for 2.7.

The [suggested solution](https://packaging.python.org/guides/dropping-older-python-versions/) to this problem (requiring 2.7 or 3.x+) is to first require 2.7+ and then exclude whatever 3.x versions are unsupported, such as `'>2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*'` if doing 2.7 and 3.4+.

I'm assuming based on your reply that the first sub-version of 3 that is compatible is 3.6, so I'll go ahead and implement the above method and mark this again.

Reply:
One more reference for good measure https://twitter.com/hynek/status/963711338500558850

Reply:
I am not asking how commas work. I am asking what the point line 14 is.

Reply:
That line isn't necessary with the newer solution I found and in fact was already removed in my most recent commit.